PLAN.md

```
@@ -0,0 +1,962 @@
+| `idle` | Not yet triggered (trigger ≠ "immediate") |
+| `loading` | All children mounting, none resolved yet — fallback shown |
+| `partial` | Some children resolved (progressive mode only) — resolved children visible, others loading |
+| `partial-error` | Some resolved, some failed — resolved children visible, failed children show error UI |
+| `resolved` | All children resolved — full content visible |
+| `error` | All children failed — error UI shown |
+| `refreshing` | `staleWhileRefresh=true`, refresh in progress — stale content + `suspense-refreshing` class |
+| `refresh-error` | `staleWhileRefresh=true`, refresh failed — error UI replaces stale content |
+
+### Transitions
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)